x-pack/test/accessibility/apps/dashboard_edit_panel.ts

```diff
@@ -33,100 +34,68 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
+    it('can view more actions', async () => {
+      await dashboardPanelActions.openContextMenuMorePanel();
       await a11y.testAppSnapshot();
     });
 
-    // https://github.com/elastic/kibana/issues/77422
-    it.skip('A11y test on dashboard embeddable custom time range', async () => {
-      await testSubjects.click('embeddablePanelToggleMenuIcon');
+    it('can create a custom time range', async () => {
+      await dashboardPanelActions.openContextMenuMorePanel();
       await testSubjects.click('embeddablePanelAction-CUSTOM_TIME_RANGE');
       await a11y.testAppSnapshot();
+      await testSubjects.click('addPerPanelTimeRangeButton');
     });
 
-    // flow will change whenever the custom time range a11y issue gets fixed.
-    // Will need to click on gear icon and then click on more.
-
-    // inspector panel
-    it('A11y test on dashboard embeddable open inspector', async () => {
-      await testSubjects.click('embeddablePanelAction-openInspector');
+    it('can open inspector', async () => {
+      await dashboardPanelActions.openInspector();
       await a11y.testAppSnapshot();
       await testSubjects.click('euiFlyoutCloseButton');
     });
 
-    // fullscreen
-    it('A11y test on dashboard embeddable fullscreen', async () => {
-      await testSubjects.click('embeddablePanelToggleMenuIcon');
-      await testSubjects.click('embeddablePanelMore-mainMenu');
-      await testSubjects.click('embeddablePanelAction-togglePanel');
-      await a11y.testAppSnapshot();
-    });
-
-    // minimize fullscreen panel
-    it('A11y test on dashboard embeddable fullscreen minimize ', async () => {
-      await testSubjects.click('embeddablePanelToggleMenuIcon');
-      await testSubjects.click('embeddablePanelMore-mainMenu');
-      await testSubjects.click('embeddablePanelAction-togglePanel');
+    it('can go fullscreen', async () => {
+      await dashboardPanelActions.clickExpandPanelToggle();
       await a11y.testAppSnapshot();
+      await dashboardPanelActions.clickExpandPanelToggle();
     });
 
-    // replace panel
-    it('A11y test on dashboard embeddable replace panel', async () => {
-      await testSubjects.click('embeddablePanelToggleMenuIcon');
-      await testSubjects.click('embeddablePanelMore-mainMenu');
-      await testSubjects.click('embeddablePanelAction-replacePanel');
+    it('can replace panel', async () => {
+      await dashboardPanelActions.replacePanelByTitle();
       await a11y.testAppSnapshot();
       await testSubjects.click('euiFlyoutCloseButton');
     });
 
-    // delete from dashboard
-    it('A11y test on dashboard embeddable delete panel', async () => {
-      await testSubjects.click('embeddablePanelToggleMenuIcon');
-      await testSubjects.click('embeddablePanelMore-mainMenu');
-      await testSubjects.click('embeddablePanelAction-deletePanel');
+    it('can delete panel', async () => {
+      await dashboardPanelActions.removePanel();
```

Comment:
Use team-maintained page objects if available because sometimes "removing a panel" is more complicated than it seems